targon/internal/targon/callbacks.go

```diff
@@ -386,7 +373,7 @@ func setWeights(v *boilerplate.BaseChainSubscriber, c *Core, h types.Header) {
 
 	go func() {
 		color := "3447003"
-		title := fmt.Sprintf("Validator setting weights at block %v", h.Number)
+		title := fmt.Sprintf("Validator setting weights at block %v", h.Number) // chnage to get weights
```

Comment:
1) why is this comment here
2) This entire go func can be moved to inside the callback and not inside setweights now that getweights is no longer internal to `setWeights`

Reply:
This also means you can remove the header hash from the inputs of setweights